proposed/0000-Use-public-key-for-id-on-all-messages.md

```diff
@@ -0,0 +1,60 @@
+- Feature Name: Use public key alone to authorise actions 
+- Type Enhancement
+- Related components routing, types, client, vaults
+- Start Date: 23-06-2015
+- RFC PR: 
+- Issue number: 
+
+# Summary
+
+At the moment client Id packets are create and a hash of these packets used as identity. 
+This involves storing such packets prior to the client interacting with the network. 
+As such this requires an initial `unauthorised put` as the client is not known to the 
+network and cannot be recognised without this. It also means there are many lookups
+to link the hash of the id packet with the client taking the action. 
+These lookups are costly it is proposed here that they are not required.  
+
+# Motivation
+
+Storing Id packets is a strain on the network. Lookups are a strain and will slow down
+network actions. Id packets themselves may be a security issue (can they be hacked), although
+highly unlikely, it is impossible to hack if they do not exist! 
+
+This mechanism does not reduce the total number of clients or types of clients as they all exist 
+in an address space of 2^256 and app developers can use multiple Id's in very clever ways, such 
+as the SAFE network does to distinguish public id's and private id's. Now these and more are possible.
```

Comment:
Are you agreeing with the rfc and the comment I made on it? confused! Ah I think I See what you are saying, the work `not` is missing :-)